### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by lisandroseia**
In this branch I made:
A new contact section with a form
Styled the contact section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -271,6 +272,50 @@ <h3 class="ab-me-title">Skills</h3>
...
+            />
+
+            <input
+              class="work-child"
+              type="email"
+              name="email"
+              placeholder="Enter your email"
+              required
+            />
+
```
Comment (andersonlebon) [inline index.html:302]: - [ ] ***CONGRATULATION*** :clap: p: your contact form is working properly. However one of the requirements says that you should stick to the design templates in Figma as much as possible, this includes the font, colors, images, and spacing (margin/padding). Now compare your design on the right and the design of the template on the left of the image :point_down:
Screenshot from 2021-12-01 00-15-40
 Kindly follow the template design and improve your contact form.

**Review by andersonlebon — CHANGES_REQUESTED** [review]
Hi @lisandroseia 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (lisandroseia) [reply index.html:302]: Hello, I have been trying to download the correct image for an hour but it seems to be impossible, do you know who could help me with this?

File: assets/styles.css [hunk 2]
```diff
@@ -343,3 +344,106 @@ h1 {
...
+
+#container4 {
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  width: 90%;
+  height: 100%;
+  align-content: center;
+  justify-content: space-between;
+}
```
Comment (andersonlebon) [inline assets/styles.css:375]: ## [OPTIONAL]
- [ ] Kindly I would suggest working on the space between the `contact-header` and the `form`

**Review by andersonlebon — CHANGES_REQUESTED** [review]
### Hi @lisandroseia 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️
- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alonsofalconi — APPROVED** [review]
Hi @lisandroseia and @sumairq,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lisandroseia** [comment]
Thanks for all the reviews, I added very useful stuff to my projects

**Pull request merged**